4001-5000/LIT4958Priests.xml

```diff
@@ -1,12 +1,13 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng" 
-schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng" 
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng"
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT4958Priests" xml:lang="en" type="work">
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title xml:lang="gez" xml:id="t1">ድርሳን፡ በእንተ፡ ክብሮሙ፡ ለ፳፬፡ ካህናተ፡ ሰማይ፡</title>
-                <title xml:lang="gez" corresp="#t1" type="normalized">Dǝrsān baʾǝnta kǝbromu la-24 kāhǝnāta samāy</title>
-                <title xml:lang="en" corresp="#t1">Homily on the glory of the 24 Heavenly Priests</title>
+              <title xml:lang="en" corresp="#t1">Homily on the glory of the 24 Heavenly Priests by Cyril, Bishop of Jerusalem</title>
```

Comment:
If "Homily on the glory of the 24 Heavenly Priests by Cyril, Bishop of Jerusalem" is the title for the article, I would suggest:
```
                <title xml:lang="gez" xml:id="t1">ድርሳን፡ በእንተ፡ ክብሮሙ፡ ለ፳፬፡ ካህናተ፡ ሰማይ፡</title>
                <title xml:lang="gez" corresp="#t1" type="normalized">Dǝrsān baʾǝnta kǝbromu la-24 kāhǝnāta samāy</title>
                <title xml:lang="en" corresp="#t1">Homily on the Glory of the 24 Heavenly Priests</title>
<title xml:lang="en" xml:id="t2" type="main">Homily on the glory of the 24 Heavenly Priests by Cyril, Bishop of Jerusalem</title>
```

Reply:
Yes. But I would prefer t1 English followed by t2 Ethiopic (so, title 1 is not the exact translation of title 2). With homilies, it is always better to provide an author, it he is mentioned. Will it come out like this?

Reply:
Yes, without the "", because it is not a translation anymore

Reply:
Sorry, do not understand. No "" - where?

Reply:
Sorry, in the PDF I mean